molecule/default/molecule.yml

```diff
@@ -3,30 +3,57 @@ dependency:
   name: galaxy
 driver:
   name: docker
-lint:
-  name: yamllint
+lint: |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
 platforms:
-  - name: debian9_systemd
-    image: geerlingguy/docker-debian9-ansible:latest
+  - name: debian10_systemd
+    image: geerlingguy/docker-debian10-ansible:latest
     privileged: yes
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
     command: /lib/systemd/systemd
     pre_build_image: yes
-  - name: debian10_systemd
-    image: geerlingguy/docker-debian10-ansible:latest
+  - name: debian11_systemd
+    image: cisagov/docker-debian11-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+  - name: kali_systemd
+    image: cisagov/docker-kali-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+  - name: fedora32_systemd
+    image: geerlingguy/docker-fedora32-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+  - name: fedora33_systemd
+    image: cisagov/docker-fedora33-ansible:latest
     privileged: yes
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
     command: /lib/systemd/systemd
     pre_build_image: yes
```

Comment:
Was there a reason for leaving out Ubuntu?  Not a big deal, but Ubuntu usually comes along for free with Debian.

Reply:
> Was there a reason for leaving out Ubuntu? Not a big deal, but Ubuntu usually comes along for free with Debian.

When `molecule` was installing `pip` (a dependency of `docker`, which is a dependency for `ansible-role-pca-gophish-composition`), I was getting this error last night for both Ubuntu distributions and I wasn't sure why:
```
    TASK [pip : Install pip] *******************************************************
fatal: [ubuntu1604_systemd]: FAILED! => {"changed": false, "msg": "No package matching 'python3-pip' is available"}
fatal: [ubuntu1804_systemd]: FAILED! => {"changed": false, "msg": "No package matching 'python3-dev' is available"}
```

If you have any ideas, please let me know.  I thought it was odd because this functionality works fine when I `molecule test` `ansible-role-docker`.

Reply:
Just to be sure, did you add the Ubuntu `apt` cache update in `prepare.yml` per https://github.com/cisagov/ansible-role-docker/blob/develop/molecule/default/prepare.yml#L12 ?

Reply:
Ah, yes, I think that you are on to something there @mcdonnnj - will put that in and report back...

Reply:
That was definitely it - thanks @mcdonnnj!  Fixed in https://github.com/cisagov/ansible-role-pca-gophish-composition/pull/1/commits/e5b153eb14bceb2436f6cc8f1acf76ccd720991c and https://github.com/cisagov/ansible-role-pca-gophish-composition/pull/1/commits/53631fcb5fc80c1803b25e7b38cbfc7b13facc2e.